src/elm/Page/Dashboard/Claim.elm

```diff
@@ -303,6 +354,90 @@ update msg model _ =
                 |> UR.init
                 |> UR.logGraphqlError msg err
 
+        OpenModal claimId vote ->
+            { model | modalStatus = ModalOpened claimId vote } |> UR.init
+
+        CloseModal ->
+            { model | modalStatus = ModalClosed } |> UR.init
+
+        VoteClaim claimId vote ->
+            case model.statusClaim of
+                Loaded _ ->
+                    let
+                        newModel =
+                            { model
+                                | modalStatus = ModalLoading claimId vote
+                            }
+                    in
+                    if LoggedIn.isAuth loggedIn then
+                        UR.init newModel
+                            |> UR.addPort
+                                { responseAddress = msg
+                                , responseData = Encode.null
+                                , data =
+                                    Eos.encodeTransaction
+                                        [ { accountName = loggedIn.shared.contracts.community
+                                          , name = "verifyclaim"
+                                          , authorization =
+                                                { actor = loggedIn.accountName
+                                                , permissionName = Eos.samplePermission
+                                                }
+                                          , data = Claim.encodeVerification claimId loggedIn.accountName vote
+                                          }
+                                        ]
+                                }
+
+                    else
+                        UR.init newModel
+                            |> UR.addExt (Just (VoteClaim claimId vote) |> LoggedIn.RequiredAuthentication)
+
+                _ ->
+                    model
+                        |> UR.init
+
+        GotVoteResult claimId (Ok _) ->
+            case model.statusClaim of
+                Loaded claim ->
+                    let
+                        message val =
+                            [ ( "value", val ) ]
+                                |> I18Next.tr loggedIn.shared.translations I18Next.Curly "claim.reward"
+
+                        value =
+                            String.fromFloat claim.action.verifierReward
+                                ++ " "
+                                ++ Eos.symbolToString claim.action.objective.community.symbol
+                    in
+                    { model
+                        | statusClaim = Loading
+                    }
+                        |> UR.init
+                        |> UR.addExt (LoggedIn.ShowFeedback LoggedIn.Success (message value))
```

Comment:
Github lost my review here 😢 

So after voting I'm presented to this screen:

<img width="1327" alt="Screen Shot 2020-10-20 at 18 35 46" src="https://user-images.githubusercontent.com/1082127/96647462-164eec00-1304-11eb-93bd-12e6e45c02f0.png">

The feedback is with an weird spacing and the buttons go into a loading state. The vote itself works fine!

Reply:
@lucca65 Should be fixed now.